### ISSUE TITLE
Model of the Month

### ISSUE THREAD

**Issue by rsignell-usgs** [ioos/catalog#40]
Let's try to get one set of models each month fully available via [the IOOS Catalog](https://data.ioos.us/), including:
* discoverable via CSW with bounding box and time extents
* WMS endpoints enabled (this may involve setting up ncWMS2 services for unstructured grid data)
* Appropriate `title` and `summary`

Here's the list so far:

- [x] March:  MARACOOS Espresso ([success screenshot](https://github.com/ioos/catalog/issues/31#issuecomment-292513867))
- [x] April: NERACOOS NECOFS ([successful harvest](https://github.com/neracoos-open/DMAC/issues/1#issuecomment-292741786), [successful screenshot](https://github.com/neracoos-open/DMAC/issues/1#issuecomment-293216255))
- [ ] May: SECOORA NCSTATE
- [ ] June: GCOOS 
- [ ] July: 
- [ ] August:
- [ ] September:
- [ ] October:
- [ ] November:
- [ ] December:
- [ ] January:
- [ ] February:

Need to make sure:
*  Model runs are compliant with [CF](http://cfconventions.org/), [UGRID](http://ugrid-conventions.github.io/ugrid-conventions/) and [SGRID](http://sgrid.github.io/sgrid/) conventions.
*  Datasets are being aggregated.
*  Aggregations have ISO services enabled with up-to-date ncISO.
*  ISO records are being harvested into local WAF.   See this [Docker-containerized crawler](https://github.com/ioos/catalog/blob/master/waf_generation_scripts/docker_thredds_iso_crawler/Readme.md) for an easy way to harvest ISO records from THREDDS. 
* WAFs of ISO data are configured for harvest in the [IOOS Registry](https://registry.ioos.us/harvests).
* IOOS Registry API is used to force harvest after ISO records are created.
* for UGRID data, ncWMS2 services are enabled and configured.
* If WMS services are coming from services external to THREDDS, THREDDS catalogs need to be modified.
* THREDDS and ncWMS2 services are up-to-date.
* Provides are encouraged to switch to Docker containers for [THREDDS](https://hub.docker.com/r/unidata/thredds-docker/) and [ncWMS2](https://github.com/axiom-data-science/docker-ncwms) to provide services.
* Demonstrate that WMS services can be discovered and accessed through the catalog by creating a dynamic CSW catalog search in TerriaJS and displaying the results (e.g. [this NERACOOS example](https://github.com/neracoos-open/DMAC/issues/1#issuecomment-293219002)).

**Comment by mwengren** [comment]
@rsignell-usgs thanks for doing this.  We should put you on the agenda for an upcoming DMAC webinar, perhaps June after the initial meeting in early May after you've worked on the SECOORA model(s)?

**Comment by rsignell-usgs** [comment]
@mwengren would be a pleasure to give an update on this...

**Comment by mwengren** [comment]
I think the long term plan to include RA models in the Catalog will be to go through the EDS THREDDS server (when available), which is intended to include all we know of.

RPS will attribute this properly so there are references back to the RA source datasets in the THREDDS/ISO attribution (for continuity purposes and to direct users that want to get data from the source there), but if the RAs themselves want to publish their regional models in the Catalog, it will be up to them to manage it.  Unless @rsignell-usgs  wants to take this back up.  Closing for now.

**Comment by rsignell-usgs** [comment]
This issue makes me sad.  It would have been (and still would be) super useful, and achievable, but I just got distracted/pulled on to other things.  Having stuff on the EDS THREDDS is not really the same.   I'd rather keep this open.

**Comment by mwengren** [comment]
I agree this would be nice to do.  We have limited bandwidth for what we can do on the Catalog project itself however, so it will take a group effort.  

Instead of letting this linger, why don't we set a goal to add the known RA models that aren't already there by an upcoming Catalog milestone?  I'll set this for Release 1.6 due at the end of June.

We can get @ocefpaf's help in case there are any updates needed to this notebook: https://ioos.github.io/notebooks_demos/notebooks/2018-12-04-grids/.  This is the best list of RA models that I know of, other than looking at eds.ioos.us.  Ideally, they both are in sync.

What if we update the checklist based on that (I added UW LiveOcean since that is a separate model not included in the notebook, and a CARICOOS placeholder): 

- [x] MARACOOS Doppio

- [x] MARACOOS NYHOPS

- [x] NERACOOS NECOFS

- [x] SECOORA CNAPS

- [x] NANOOS SELFE

- [x] NANOOS OSU ROMS

- [ ] PacIOOS Hawaii ROMS

- [ ] WestCoastUCSC (SCCOOS?)

- [ ] CARICOOS SWAN (missing from Data Demo Center)

- [ ] ...

Hold for now:

- [ ] WCOFS  (non operational, should not be included in Catalog)

- [ ] UW LiveOcean (No public THREDDS server so shouldn't be in Catalog, missing from Data Demo Center)

Any others out there that should go on this list?  I know there are some CARICOOS models that aren't yet in the Catalog, but don't know their names and they aren't in the Demo Center notebook either.

@rsignell-usgs We have the March and May timeslots open for DMAC tech webinars.  Can you look into following the process you listed with an RA data manager to add one of these models and possibly give a demo of the results during either of those two slots?  Hopefully others will follow suit with any more missing models and this can be closed by June.

**Comment by kbailey-noaa** [comment]
CARICOOS source: http://dm1.caricoos.org/thredds/catalog.html

**Comment by benjwadams** [comment]
@mwengren:  This extension looks like it'd probably do the trick: https://github.com/ckan/ckanext-showcase
